.github/workflows/release.yaml

```diff
@@ -39,3 +43,20 @@ jobs:
         env:
           CR_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
           CR_GENERATE_RELEASE_NOTES: true
+
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v3
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Push charts to GHCR
+        run: |
+          shopt -s nullglob
+          for pkg in .cr-release-packages/*; do
+            if [ -z "${pkg:-}" ]; then
+              break
+            fi
+            helm push "${pkg}" "oci://ghcr.io/${GITHUB_REPOSITORY}"
```

Comment:
@M0NsTeRRR can you run another test in your fork with this update.

Reply:
As I said in the other thread I can't run it as my github name as uppercase leading in invalid repository
https://github.com/M0NsTeRRR/opentelemetry-helm-charts/actions/runs/12046348284/job/33586644364